reset_user_access_right/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+========================
+Reset User Access Right
+========================
```

Comment:
You should remove one '=' (bottom and top) just like
===================
Reset User Access Right
===================
(ref https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)